getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -851,15 +851,15 @@ Basic syntax::
 
 1. Replace ``switch`` with ``match``.
 2. Remove ``case``.
-3. Remove any ``break``\ s. If you don't want to ``break`` by default, you can use ``continue`` for a fallthrough.
+3. Remove any ``break``\ s. If you don't want to ``break`` by default, you can use ``continue`` followed by a Multiple pattern for a fallthrough.
```

Comment:
Why is "Multiple" capitalized like this? This change does not make this statement any more clear to me.

Reply:
It's in reference to the "Multiple" pattern type described in the following examples. Perhaps "followed by an additional matching pattern" would be more clear?

Reply:
I see the "Multipatterns" section below the specific pattern examples. Is that what you are referring to?

To be honest the wording "continue followed by..." makes it sound like there might be parameters to the `continue` keyword, as if it were a method or something.

Reply:
Interesting... so that is what I'm referring to, but that part is different between the live site and the version here on github that I read while editing. It doesn't match any version of this document, and I don't see any issues or PRs related to the differences. Why would that be?

Considering the live document, this line might more properly be something like "you can use continue and include a multipattern that also matches [the expression?] for a fallthrough", yes?

Reply:
In general, always prefer making changes against the `master` branch, and only care about that branch. Only exceptions are if your changes are only relevant for specific, older versions.